### PR TITLE
Ivan_Vasilev_lab_1_var_3

### PR DESCRIPTION
**Pull request by AdAndRoll**

### CODE REVIEW

**Comment by AdAndRoll** [comment]
test passed

File: clang/labs/lab1/vasilev_ivan/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+add_llvm_library(DeprecationPlugin MODULE Find_Deprication.cpp PLUGIN_TOOL clang)
```
Comment (m-ly4) [inline clang/labs/lab1/vasilev_ivan/CMakeLists.txt]:
```suggestion
add_llvm_library(DeprecationPlugin MODULE Find_Deprecation.cpp PLUGIN_TOOL clang)
```
Please fix this typo in all files names.

File: clang/labs/lab1/vasilev_ivan/Find_Deprication.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  std::unique_ptr<clang::ASTConsumer>
+  CreateASTConsumer(clang::CompilerInstance &Compiler,
+                    llvm::StringRef InFile) override {
+    return std::make_unique<DeprecationWarnConsumer>();
+  }
+
+protected:
+  bool ParseArgs(const clang::CompilerInstance &Compiler,
+                 const std::vector<std::string> &args) override {
+    return true;
```
Comment (m-ly4) [inline clang/labs/lab1/vasilev_ivan/Find_Deprication.cpp]: Could you please add an option that turns on search case insensitive `deprecated` substring in names?

**Review by m-ly4 — CHANGES_REQUESTED** [review]

**Comment by aobolensk** [comment]
Please, fix PR name: lab 1, variant 3

**Review by m-ly4 — APPROVED** [review]

**Pull request merged**